python/sglang/srt/model_executor/model_runner.py

```diff
@@ -412,18 +412,70 @@ def model_specific_adjustment(self):
             self.init_double_sparsity_channel_config(server_args.ds_heavy_channel_type)
 
         if self.is_multimodal:
-            self.mem_fraction_static *= 0.90
-            logger.info(
-                f"Automatically reduce --mem-fraction-static to {self.mem_fraction_static:.3f} "
-                f"because this is a multimodal model."
-            )
             if not self.is_multimodal_chunked_prefill_supported:
                 server_args.chunked_prefill_size = -1
                 logger.info(
-                    f"Automatically turn of --chunked-prefill-size as it is not supported for "
+                    f"Automatically turn off --chunked-prefill-size as it is not supported for "
                     f"{self.model_config.hf_config.model_type}"
                 )
 
+            # roughly reduce the mem_fraction_static base on params of Vit
```

Comment:
move this to `server_args.py` https://github.com/Alcanderian/sglang/blob/562f279a2d802e14cb3cf7bd55cae559f467dd9a/python/sglang/srt/server_args.py#L284